TypeSystem/Serialization/test.cc

```diff
@@ -60,6 +60,14 @@ CURRENT_STRUCT(Serializable) {
   bool operator<(const Serializable& rhs) const { return i < rhs.i; }
 };
 
+CURRENT_STRUCT(Int) {
+  CURRENT_FIELD(x, int32_t, 0);
```

Comment:
I take issue with the above two comments. Whiskey and drunk, while not necessarily orthogonal, are not identical.